app/lib/navigation/state_file_nj_question_navigation.rb

```diff
@@ -75,6 +78,13 @@ def self.show_progress?(controller_class)
                                           Navigation::NavigationStep.new(StateFile::Questions::ReturnStatusController),
                                         ], false),
     ].freeze
-    FLOW = SECTIONS.map(&:controllers).flatten.freeze
+
+    def self.controllers
+      sections.flat_map(&:controllers)
+    end
+
+    def self.pages(visitor_record)
+      sections.flat_map { |section| section.pages(visitor_record) }
+    end
```

Comment:
can the methods be refactored to be defined once or is this b/c we need these to be class methods specifically on these navigations?

Reply:
these have to be here so that they're defined _after_ (and therefore override) the versions defined in the module that's included at the top of this file. barring a larger refactor, they have to be here